.github/workflows/release.yml

```diff
@@ -357,41 +357,38 @@ jobs:
         id: parse_version
         run: |
           VERSION=$(curl -s -H "Authorization: token ${{ github.token }}" \
-            https://api.github.com/repos/${{ github.repository }}/releases | \
-            jq -r "[.[] | select(.body | contains(\"- jobId: ${{ github.run_id }}\\n\"))] | first | .tag_name")
+                      https://api.github.com/repos/${{ github.repository }}/releases | \
+                      jq -r "[.[] | select(.body | contains(\"- jobId: ${{ github.run_id }}\\n\"))] | first | .tag_name")
           echo ::set-output name=version::${VERSION}
 
-      - name: saucectl test
+      - name: cloud tests (Windows)
         working-directory: ./tests/fixtures/post-release
+        env:
+          BUILD_ID: Github RUN ID ${{ env.GITHUB_RUN_ID }}
+        run: |
+          npx saucectl run --runner-version "url: https://github.com/saucelabs/sauce-testcafe-runner/releases/download/${{ steps.parse_version.outputs.version }}/sauce-testcafe-runner-win-amd64.zip" --config ./.sauce/config_win.yml 
+
+  post-release-macos-tests:
+    runs-on: ubuntu-latest
+    needs: publish-release
+    env:
+      SAUCE_ACCESS_KEY: ${{secrets.SAUCE_ACCESS_KEY}}
+      SAUCE_USERNAME: ${{secrets.SAUCE_USERNAME}}
+    steps:
+      - uses: actions/checkout@master
+      - name: Install Deps
+        run: npm ci
```

Comment:
Why do we need to install dependencies at this point? 🤔

Reply:
Oh, because of `saucectl` itself.

Reply:
Imho, it'd be better if you just install `saucectl` on the fly and nothing else at this point. Faster certainly.

Reply:
sounds better.